FinniversKit/Sources/Components/SelectionListView/SelectionListView.swift

```diff
@@ -20,6 +20,14 @@ public class SelectionListView: UIView {
         itemViews.contains(where: { $0.isSelected })
     }
 
+    public var selectedItems: [(index: Int, identifier: String?)] {
+        itemViews.enumerated().map { index, itemView in
+            guard itemView.isSelected else { return nil }
+
+            return (index: index, identifier: itemView.model.identifier)
+        }.compactMap { $0 }
```

Comment:
This could be simplified:

```swift
itemViews.enumerated().compactMap { index, itemView in
    guard itemView.isSelected else { return nil }

    return (index: index, identifier: itemView.model.identifier)
}
```